core-bundle/src/Twig/Runtime/CspRuntime.php

```diff
@@ -42,6 +42,10 @@ public function inlineStyles(string $htmlFragment): string
             return $htmlFragment;
         }
 
+        if (!$styles = $this->wysiwygProcessor->extractStyles($htmlFragment)) {
+            return $htmlFragment;
+        }
+
```

Comment:
```suggestion
```

Looks like a failed rebase/merge as this code block is the same as the one above.